### PR TITLE
Fix `powerMonitor` timeout bugs and add extra logs

### PR DESCRIPTION
**Pull request by john-farina**
## The Problem
Brian ran into [This Bug](https://swivvel.slack.com/archives/C041MFH7VMF/p1697127153690059), which came from the recent change in `handlePowerMonitorStateChanges` - Where I set it up to close the windows after 10 minutes. 

But if the window closed and we still had the timeout, the `transparentWindow` never opened. Causing a state where we can't get that open again

## The Solution
Always open the windows even if there is a `timeout`. Since if the window already exists it will use that one

## Other Changes
Add extra logging with more information so we can properly debug this if another error occurs.

### CODE REVIEW

File: src/background/handlePowerMonitorStateChanges.ts [hunk 1]
```diff
@@ -44,8 +44,8 @@ export default (state: State, windowService: WindowService): void => {
```
Comment (cedric-swivvel) [inline src/background/handlePowerMonitorStateChanges.ts:44]: I've found logging to be most helpful in the desktop app when it is extremely verbose. I'd probably add even more log lines here:

```ts
if (timeout) {
  log.info(`Power monitor: lock-screen: window close timeout already exists; clearing timeout`)
  clearTimeout(timeout);
}

log.info(`Power monitor: lock-screen: setting new window close timeout...`)
timeout = setTimeout(() => {
  log.info(`Power monitor: lock-screen: window close timeout reached`)
  windowService.closeAllWindows();
  log.info(`Power monitor: lock-screen: setting window close timeout to null`)
  timeout = null
}, TEN_MIN_MS);
```

We don't need to add log lines for things that already do logging (e.g. `closeAllWindows()` probably already has log lines saying what is going on), but with the additional log lines above it'll be easier to know exactly what the app is doing when we read the logs.

File: src/background/handlePowerMonitorStateChanges.ts [hunk 2]
```diff
@@ -57,27 +57,29 @@ export default (state: State, windowService: WindowService): void => {
...
-
       timeout = null;
+      log.info(`Power monitor suspend (timeout): closed all windows`);
     }, TEN_MIN_MS);
   });
   powerMonitor.on(`unlock-screen`, () => {
     log.info(`Power monitor: unlock-screen detected`);
 
     if (timeout) {
+      log.info(`Power monitor: clearing timeout`);
```
Comment (cedric-swivvel) [inline src/background/handlePowerMonitorStateChanges.ts:68]: I liked your idea of including the even name in the log line to make it more clear. Want to do that here too?

```ts
log.info(`Power monitor: unlock-screen: clearing timeout`);
```

Comment (cedric-swivvel) [reply src/background/handlePowerMonitorStateChanges.ts:68]: I know that we log the event name right above it, but it's already pretty hard to follow the logs with the level of verbosity that we use, so I think that anything that can provide a little extra context on a single line can be helpful when trying to understand what's going on.

Comment (john-farina) [reply src/background/handlePowerMonitorStateChanges.ts:68]: Sounds good!

**Review by cedric-swivvel — APPROVED** [review]
